rose/server/game.py

```diff
@@ -92,13 +96,9 @@ def drive_player(self, name, info):
         self.players[name].response_time = info.get('response_time', 1.0)
 
     def print_stats(self):
-        print
-        print 'Stats'
+        log.info('Stats')
         for i, p in enumerate(sorted(self.players.values())):
-            print '%d  %10s  row:%d  score:%d' % (
-                i+1, p.name, p.y, p.score
-            )
-        print
+            log.info('%d  %10s  row:%d  score:%d', i + 1, p.name, p.y, p.score)
```

Comment:
I think we want to format a multiline string and log it, so the message will look like:

```
   game:INFO:Stats
       1   player1  row:0  score:600
       2   player2  row:1  score:590
```